addons/mail/static/src/core/web/recipients_input_tags_list.js

```diff
@@ -0,0 +1,50 @@
+import { usePopover } from "@web/core/popover/popover_hook";
+import { TagsList } from "@web/core/tags_list/tags_list";
+import { RecipientsInputTagsListPopover } from "./recipients_input_tags_list_popover";
+
+import { useEffect } from "@odoo/owl";
+
+export class RecipientsInputTagsList extends TagsList {
```

Comment:
Missing docstring explaining what it does ^^ (otherwise we end up with a lot of small stuff we don't exactly know what they do, unless reading the whole code)